R/git.R

```diff
@@ -357,7 +357,8 @@ git_sitrep <- function(tool = c("git", "github"),
     if (!vaccinated) {
       ui_bullets(c("i" = "See {.fun usethis::git_vaccinate} to learn more."))
     }
-    kv_line("Default Git protocol", git_protocol())
+    protocol <- suppressMessages(git_protocol())
```

Comment:
I think `ui_silence()` is probably our official way of doing this. And it may be better to do that, so that we only silence output coming from usethis itself. It probably doesn't really matter in practice, but it is what I would do here, I think, so let's try it.